.eslintrc.js

```diff
@@ -0,0 +1,126 @@
+module.exports = {
```

Comment:
Let's not increase the complexity of the project right now & stick to what Next.js provides by default. So, could you revert back to the older `.eslintrc.json`? We'll review the configurations after the first iteration of the landing page is complete.

Reply:
code will be messy, and i don't want that.